addons/website/views/snippets/s_timeline_list.xml

```diff
@@ -0,0 +1,101 @@
+                        <span class="o_dot o_not_editable position-absolute translate-middle-x rounded-circle pe-none text-o-color-1" contenteditable="false"/>
+                        <small class="text-muted">Oct 09, 2024</small>
+                        <b>Mobile App Compatibility Expanded</b>
+                    </div>
+                    <div class="s_timeline_list_row position-relative pb-4" data-name="Milestone">
+                        <div class="o_dot_line position-absolute top-0 bottom-0 w-0 mb-1 border-start pe-none"/>
+                        <span class="o_dot o_not_editable position-absolute translate-middle-x rounded-circle pe-none text-o-color-1" contenteditable="false"/>
+                        <small class="text-muted">Dec 22, 2024</small>
+                        <b>Advanced Analytics Tools Introduced</b>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="s_timeline_list_options" inherit_id="website.snippet_options">
+    <xpath expr="//t[@t-call='website.snippet_options_background_options']" position="before">
+        <div data-js="MultipleItems" data-selector=".s_timeline_list">
+            <we-row string="Milestones">
+                <we-button data-add-item="" data-placeholder-template="website.s_timeline_list.milestonePlaceholder" data-placeholder-container=".s_timeline_list_wrapper > div" data-item=".s_timeline_list_row:first" data-select-item="" data-add-before="true" data-no-preview="true" class="o_we_bg_brand_primary">
+                    Add New
+                </we-button>
+            </we-row>
+        </div>
+    </xpath>
+    <xpath expr="." position="inside">
+        <div data-selector=".s_timeline_list_row" data-drop-near=".s_timeline_list_row"/>
+        <div data-selector=".s_timeline_list">
+            <we-button-group string="Alignment" title="Alignment" data-apply-to=".s_timeline_list_wrapper">
+                <we-button title="Align Left"
+                        data-select-class="justify-content-start"
+                        class="fa fa-align-left"/>
+                <we-button title="Align Center"
+                        data-select-class="justify-content-center"
+                        class="fa fa-align-center"/>
+                <we-button title="Align Right"
+                        data-select-class="justify-content-end"
+                        class="fa fa-align-right"/>
+            </we-button-group>
+        </div>
+    </xpath>
+    <xpath expr="//t[@t-set='o_dot_color_selector']" position="attributes">
+        <attribute name="data-selector" add=".s_timeline_list .s_timeline_list_row" separator=", "/>
+    </xpath>
+    <xpath expr="//t[@t-set='o_dot_line_color_selector']" position="attributes">
+        <attribute name="data-selector" add=".s_timeline_list" separator=", "/>
+    </xpath>
```

Comment:
- tested on runbot and the options are not there :thinking:

![image](https://github.com/user-attachments/assets/13aca194-bf25-4341-b3b3-33ea52d510fb)

Reply:
> Why xpath o_dot_color_selector and o_dot_line_color_selector for s_timeline_list when you don't for the s_timeline snippet? I think both could be xpathed

I proceeded this way just to avoid having an "empty"  `t-set` in snippets.xml. 
Having to choose, I opted for adding `s_timeline` because it's pre-existing to this PR and would make sense by looking at the commits history.

I'm ok to the change tho, do you confirm? 👇🏼

```diff
-    <t t-set="o_dot_color_selector" t-valuef=".s_timeline .s_timeline_row"/>
+    <t t-set="o_dot_color_selector"/>
```

Reply:
Ok I understand, so you can leave it like that for me... It's true that empty `t-set` looks strange, I don't see any empty `t-set` in the code.